src/encoded/docs/schema-changes.md

```diff
@@ -154,87 +154,210 @@ This new object is an array of example objects that can successfully POST agains
+
+**Specific example from the treatment object schema change:**
+For example, if we included a minor change in treatment object such that *μg/kg* could be specified as treatment units, the following test should allow one to test whether the update has been successfully implemented or not:
+
+        def test_treatment_patch_amount_units(testapp, treatment):
+            testapp.patch_json(
+            treatment['@id'],
+            {
+                'treatment_type': 'injection',
+                'amount': 20,
+                'amount_units': 'μg/kg'
+            },
+            status=200, 
+            ## Status 200 means the object was successfully patched and the schema update works as expected.
+        )
 
+5. Document the changes to the corresponding log file within the **schemas/changelogs** directory. 
+
+**Specific example from the treatment object schema change:**
+For example, a minor change in the treatment object after version 11 that allowed one to use *μg/kg* as treatment units is shown below:
+
+        ### Minor changes since schema version 11
+
+        * *μg/kg* can now be specified as amount units.
 
 ### Update schema version
 
-1. In the **schemas** directory, increment the schema version (1->2), if it meets the above rule: 
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version. 
+
+**Specific example from the genetic modifications object schema change:**
+
+Up until schema version 6 for the genetic modifications object, "validation" was an enum for the "purpose" property. As a part of schema version update to 7, "validation" was removed and instead the term "characterization" was added.
+
+
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "validation",
+                "screening",
+                "expression"
+            ]
+        },
+
+Replacing the enum "validation" by the enum "characterization" in the list of enums of the "purpose" property:
+
+        "purpose":{
+            "title": "Purpose",
+            "description": "The purpose of the genetic modification.",
+            "type": "string",
+            "enum": [
+                "activation",
+                "analysis",
+                "overexpression",
+                "repression",
+                "tagging",
+                "characterization",
+                "screening",
+                "expression"
+            ]
+        },
+
+2. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object and increment the schema version.
+
```

Comment:
Change to: "In the **schemas** directory, edit the properties in the corresponding JSON file named after the object and increment the schema version."